src/adafruit_blinka/microcontroller/generic_linux/i2c.py

```diff
@@ -20,8 +24,8 @@ def __init__(self, bus_num, mode=MASTER, baudrate=None):
             raise NotImplementedError("Only I2C Master supported!")
         _mode = self.MASTER
 
-        # if baudrate != None:
-        #    print("I2C frequency is not settable in python, ignoring!")
+        if baudrate is not None:
+            print("I2C frequency is not settable in python, ignoring!")
```

Comment:
[`warnings`](https://docs.python.org/3/library/warnings.html) is probably the best way to do it so folks can turn it off.

Reply:
Oh, I didn't know about this. Good idea.